docker/docker_image_dest.go

```diff
@@ -485,7 +485,7 @@ func (d *dockerImageDestination) PutSignatures(ctx context.Context, signatures [
 	case d.c.supportsSignatures:
 		return d.putSignaturesToAPIExtension(ctx, signatures, *instanceDigest)
 	default:
-		return errors.Errorf("X-Registry-Supports-Signatures extension not supported, and lookaside is not configured")
+		return errors.Errorf("Cannot store signature: sigstore not configured for destination registry")
```

Comment:
This should still mention `X-R-S-S`; overall the two error messages should stay in sync.